src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,38 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+
+        const digits= number.toString().split("").reverse();
+        return digits.map((digit)=>{
+            return Number(digit)
+            //return digit
+    })
+    
+        
+    } 
+
+
+
+
+
+
+
+
+
+
+
+//const newValue=number.toString().split("").reverse().join()
+
+    
+ //const newV= number.toString().split("").map((num)=>{
+    //return num
+
+    //return newV.reverse();
+
+    
+
+        
+
```

Comment:
This can be simplified a bit and all the commented out code and blank space removed. This is the exact same logic you wrote, just cleaned up a bit:
```
const numberToReversedDigits = (number) =>
    number
        .toString()
        .split("")
        .reverse()
        .map((digit) => Number(digit));
```